tests/rtc_test.py

```diff
@@ -4,7 +4,7 @@
 import datetime
 
 sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
```

Comment:
this just looks like an old hacks, you can remove all of these too

Reply:
done